### PR TITLE
IDSEQ-2515 Disable report caching

### PR DESCRIPTION
**Pull request by cdebourcy**
# Description

*The intended change, motivation and consequences.*

# Notes

*Optional observations, comments or explanations.*

# Tests

* *Server-side code should have automated tests*
* *Client-side code and scripts should have at least a manual test plan*
* *See also [IDseq PR Guidelines](https://github.com/chanzuckerberg/idseq-web/blob/master/PR_GUIDELINES.md)*

### CODE REVIEW

File: app/models/pipeline_run.rb [hunk 1]
```diff
@@ -347,6 +347,8 @@ def results_finalized?
   end
 
   def ready_for_cache?
+    # TEMPORARILY DISABLE CACHING COMPLETELY - REMOVE EARLY RETURN TO REINSTATE
+    return false
```
Comment (gregdingle) [inline app/models/pipeline_run.rb]: I'd prefer to see this as an AppConfig so we can still debug in other envs

Comment (gregdingle) [reply app/models/pipeline_run.rb]: I'd be happy to take over this PR and do that.

Comment (gregdingle) [reply app/models/pipeline_run.rb]: See https://github.com/chanzuckerberg/idseq-web/pull/3223

Comment (cdebourcy) [reply app/models/pipeline_run.rb]: Ok thanks, please feel free to do that

**Pull request closed without merging**